common/djangoapps/student/migrations/0007_auto_20170227_0530.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
As you've suggested, we could handle our gender choices at the application level and keep the models unmodified.